demos/apps/bookinfo/README.md

```diff
@@ -343,10 +341,12 @@ If you now refresh the `productpage` you'll see that while the load generator is
    $ ./cleanup.sh
    ```
 
+1. Optionally shut down the control plane services using the cleanup instructions [here](../../../kubernetes/README.md).
```

Comment:
I would remove the "Instructions subject to change" part. When we  will change it, we update.